app/models/spree/avalara_transaction.rb

```diff
@@ -22,11 +22,11 @@ def commit_avatax(invoice_dt = nil, refund = nil)
       end
     end
 
-    def commit_avatax_final(invoice_dt = nil, refund = nil)
+    def commit_avatax_final(invoice_dt = nil, refund = nil, override_tax = nil)
       if document_committing_enabled?
         if tax_calculation_enabled?
           if %w(ReturnInvoice ReturnOrder).include?(invoice_dt)
-            post_return_to_avalara(true, invoice_dt, refund)
+            post_return_to_avalara(true, invoice_dt, refund, override_tax)
```

Comment:
We don't need this in `commit_avatax` method above?

Reply:
we use `commit_avatax` only for `SalesOrder`. Our change affects only refunds i.e. `ReturnInvoice` type transactions. So we don't need it in `commit_avatax`

Reply:
We may use `ReturnOrder` also for estimation? Can we add it in both the places?